src/tool/hpcrun/cct/cct.c

```diff
@@ -68,36 +68,32 @@
 //************************* System Include Files ****************************
 
 #include <assert.h>
+#include <stdbool.h>
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
-#include <stdbool.h>
 #include <unistd.h>
 
 //*************************** User Include Files ****************************
 
-#include <memory/hpcrun-malloc.h>
+#include <hpcrun/hpcrun_return_codes.h>
 #include <hpcrun/metrics.h>
-#include <messages/messages.h>
-#include <lib/prof-lean/splay-macros.h>
-#include <lib/prof-lean/hpcrun-fmt.h>
 #include <lib/prof-lean/hpcrun-fmt.h>
 #include <lib/prof-lean/spinlock.h>
-#include <hpcrun/hpcrun_return_codes.h>
+#include <lib/prof-lean/splay-macros.h>
+#include <memory/hpcrun-malloc.h>
+#include <messages/messages.h>
 
+#include "../memory/hpcrun-malloc.h"
 #include "cct.h"
-#include "cct_addr.h"
 #include "cct2metrics.h"
-#include "../memory/hpcrun-malloc.h"
+#include "cct_addr.h"
 //#include "../ompt/ompt-interface.h"
 //#include "../memory/hpcrun-malloc.h"
 
-#define HPCRUN_CCT_KEEP_DUMMY 1
```

Comment:
This and related: seemingly unrelated change. Remove or factor into a separate commit.

Reply:
I'll take these dummy nodes related changes into another pr.